test/scales/quantitativeScaleTests.ts

```diff
@@ -0,0 +1,68 @@
+///<reference path="../testReference.ts" />
+
+describe("Scales", () => {
+  describe("QuantitativeScale", () => {
+    describe("computing extents", () => {
+      // HACKHACK #2336: QuantitativeScales don't take min/max of stringy values correctly
+      it.skip("gives the minimum and maxiumum when the domain is stringy", () => {
+        const values = ["11", "3", "2", "1"];
+        const scale = new Plottable.QuantitativeScale();
+        const computedExtent = scale.extentOfValues(values);
+
+        assert.deepEqual(computedExtent, ["1", "11"], "the extent is the miminum and the maximum values");
+      });
+
+      it("gives the minimum and maxiumum when the domain is numeric", () => {
+        const values = [11, 3, 2, 1];
+        const scale = new Plottable.QuantitativeScale();
+        const computedExtent = scale.extentOfValues(values);
+
+        assert.deepEqual(computedExtent, [1, 11], "the extent is the miminum and the maximum values");
+      });
+
+      it("ignores NaN, null, and undefined values when computing the extent", () => {
+        const values = [NaN, null, undefined, 1, 11];
+        const scale = new Plottable.QuantitativeScale();
+        const computedExtent = scale.extentOfValues(values);
+
+        assert.deepEqual(computedExtent, [1, 11], "the extent is the minimum and the maximum values");
+      });
+
+      it("ignores infinite values when computing the extent", () => {
+        const values = [Infinity, -Infinity, 1, 11];
+        const scale = new Plottable.QuantitativeScale();
+        const computedExtent = scale.extentOfValues(values);
+
+        assert.deepEqual(computedExtent, [1, 11], "the extent is the minimum and the maximum values");
+      });
+    });
+
+    describe("padding", () => {
+      it("rejects negative values for padding", () => {
+        const scale = new Plottable.QuantitativeScale();
+        assert.throws(() => scale.padProportion(-0.05), Error);
```

Comment:
I'll check the emitted message, but since this is the only `assert` in this `it()` having a message for the `assert` itself seems redundant.